w_tilde/w_tilde_curvature.py

```diff
@@ -23,48 +28,46 @@ def w_tilde_curvature_interferometer_from(
     `w_tilde_preload_interferometer_from` describes a compressed representation that overcomes this hurdles. It is
     advised `w_tilde` and this method are only used for testing.
 
+    .. math::
+        W̃_{ij} = \sum_{k=1}^N \frac{1}{n_k^2} \cos(2\pi[(g_{i1} - g_{j1})u_{k0} + (g_{i0} - g_{j0})u_{k1}])
+
     Parameters
     ----------
-    noise_map_real
+    noise_map_real : ndarray, shape (N,), dtype=float64

```

Comment:
Our current convention is to not include typing in the "Parameters" numpy style docstring.

For building `readthedocs`, I use a tool to automatically use the main function typing to perform documentation typing.

In my opinion, including the typing here repeats the typing which is fully visible in the function constructor so prefer to remove it.

Reply:
Resolve in 634037.